ja2003.appdata.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
1. did you submit these upstream
2. ideally the file should have the proper name from start.

Reply:
Well, "I am upstream". The AppData is not in the main repo. It is quite OS specific and only used for the flatpak for now.

Reply:
no, it is not used only by Flatpak. It is as specific as a .desktop file, that if you don't ship one with your Linux app, it's a bug.